openmetadata-ui/src/main/resources/ui/src/components/ActivityFeed/ActivityFeedCardV2/FeedCardFooter/FeedCardFooterNew.tsx

```diff
@@ -71,24 +76,32 @@ function FeedCardFooterNew({
                   backgroundColor: '#fde3cf',
                 }}>
                 {repliedUniqueUsersList.map((user, index) => (
-                  <div
+                  <Button
+                    className="p-0"
                     key={user}
                     style={{
                       marginLeft: index === 0 ? '0px' : '-8px',
                       zIndex: repliedUniqueUsersList.length - index,
-                    }}>
-                    <ProfilePicture
+                    }}
+                    type="text"
+                    onClick={isForFeedTab ? showReplies : undefined}>
+                    <ProfilePictureNew
                       avatarType="outlined"
                       name={user}
                       size={20}
                     />
-                  </div>
+                  </Button>
                 ))}
               </Avatar.Group>
             )}
 
             {!isPost && (
-              <ThreadIcon data-testid="thread-count" height={18} width={18} />
+              <ThreadIcon
+                data-testid="reply-count"
+                height={18}
+                width={18}
+                onClick={isForFeedTab ? showReplies : undefined}
```

Comment:
Can we use a button here and add icon in it ?

Reply:
done